src/Info.ts

```diff
@@ -162,11 +162,34 @@ export default class Info {
     }
 
     private fetchInfo(command: string): InfoData | null {
+
         const info = this.infos.find(inf => {
             return inf.command === command;
         });
 
-        if (!info) { return null; }
+        if (!info) {
+            const similarity = this.infos.slice().map(i => {
+                return {
+                    command: i.command,
+                    score: this.levenshteinDistance(command, i.command),
+                };
+            });
+
+            let message = "Did you mean: ";
+            let found = false;
+            similarity.forEach(s => {
+                if (s.score < 2) {
+                    message += "`" + s.command + "`, ";
+                    found = true;
+                }
+            });
+
+            if (found) {
+                return { message: message.slice(0, -2) + "?", counter: 0 } as InfoData;
```

Comment:
Do you really need to cast here?

Reply:
Yeah

Reply:
If you require a cast, it means something isn't set. In this case, you're missing "command"